docs/user/securing-kubeapps.md

```diff
@@ -4,77 +4,45 @@ In this guide we will explain how to secure the installation of Kubeapps in a mu
 
-Kubeapps deploys by default Tiller configuring it to listen for connections only in `localhost`. Note that one particularity of Tiller is that even if it is only listening in `localhost`, request made with the `helm` tool can still contact it because, under the hoods, what the client does is opening a [`port-forward`](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) session and connect to it to like a local connection. That means that we still need to add additional security measures to prevent unauthorized access.
-
-Since Tiller is only listening in `localhost`, for exposing Tiller functionality, Kubeapps also deploys as a [sidecar container](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns/) a [**proxy**](/cmd/tiller-proxy/README.md) that receives all the requests from the Dashboard (or any other client). This component validates the requests checking that the user is allowed to perform the requested operation and finally redirect it to Tiller.
-
 In order to take advantage of Kubeapps security features you will need to configure two things: a **TLS certificate** to control the access to Tiller and [**RBAC roles**](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to authorize requests.
 
-# Deploy Kubeapps with a TLS certificate
-
-The first step to restrict the access to Tiller is to use a TLS certificate. If we don't do this any user with access to the Pod in which Tiller is running can escalate privileges. We can generate a self-signed certificate using the tool [`openssl`](https://www.openssl.org/source/). The first thing is creating a certificate authority. You will be asked to introduce a pass phrase for that:
-
-```
-$ openssl genrsa -des3 -out kubeapps-ca.key 2048
-Generating RSA private key, 2048 bit long modulus
-.....................+++
-...................+++
-e is 65537 (0x10001)
-Enter pass phrase for kubeapps-ca.key:
-Verifying - Enter pass phrase for kubeapps-ca.key:
-```
+## Install Tiller Securely
 
-Now we can generate a root certificate with that key. You will be asked for the previous password:
+You can follow the Helm documentation for deploying Tiller in a secure way. In particular we are interested in:
 
-```
-$ openssl req -x509 -new -nodes -key kubeapps-ca.key -sha256 -out kubeapps-ca.pem -subj '/CN=localhost'
-Enter pass phrase for kubeapps-ca.key:
-```
+ - Use a TLS certificate to control the access to the Tiller deployment: https://docs.helm.sh/using_helm/#using-ssl-between-helm-and-tiller
+ - Store release info as secrets: https://docs.helm.sh/using_helm/#tiller-s-release-information
 
-Having a certificate authority we can create a proper certificate and sign it. Again you will be asked to introduce the password from the first step:
+Following those guides you can find how to create the TLS certificate that we need and the flags necessary to install Tiller:
```

Comment:
From these guides you can find out how to create the TLS certificate and the necessary flags to install Tiller securely: